recipes/wrf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0b1" %}
+
+package:
+  name: wrf-python
+  version: {{ version }}
+
+source:
+  fn: wrf-python-{{ version }}.tar.gz
+  url: https://github.com/NCAR/wrf-python/archive/{{ version }}.tar.gz
+  sha256: 94f05a9719701959702094eafb84555a996bc4f7d75042fb07bca3c35bda9f31
+
+build:
+  number: 1
+  detect_binary_files_with_prefix: true
+  skip: True  # [win32 or np>=112 or (win and py>=35)]
```

Comment:
Win32 didn't work for some reason.  I'm not sure if it's the m2w64-toolchain, numpy's distutils, or me supplying the wrong compiler argument.  It failed with a lot of missing python symbols, which should have been in libpython, so I don't think it got linked in correctly.  Since I'm a little pressed for time, I decided to skip it, and will try to figure it out after AMS.